src/HotChocolate/Core/test/Types.Tests/Types/Relay/IdAttributeTests.cs

```diff
@@ -170,13 +253,13 @@ public class Query
         {
             public string IntId([ID] int id) => id.ToString();
             public string IntIdList([ID] int[] id) =>
-                string.Join("-", id.Select(t => t.ToString()));
+                string.Join(", ", id.Select(t => t.ToString()));
             public string StringId([ID] string id) => id;
             public string StringIdList([ID] string[] id) =>
-                string.Join("-", id.Select(t => t.ToString()));
+                string.Join(", ", id.Select(t => t.ToString()));
             public string GuidId([ID] Guid id) => id.ToString();
             public string GuidIdList([ID] IReadOnlyList<Guid> id) =>
-                string.Join("-", id.Select(t => t.ToString()));
+                string.Join(", ", id.Select(t => t.ToString()));
```

Comment:
As the GUID renders with dashes it was hard to verify the list so made this a comma.